obj_init_pose: # Obj pose in some BASE_FRAME
  position: [0.7, 0.0, 0.0]
  orientation: [0.0, 0.0, 0.0, 1.0]

effector0_init_pose: # gripper pose in some BASE_FRAME
  position: [0.5, -0.12, 0.344]
  orientation: [-0.5, -0.44, -0.45, 0.59]

effector1_init_pose: # gripper pose in some BASE_FRAME
  position: [0.5, 0.13, 0.344]
  orientation: [-0.57, -0.39, -0.52, 0.50]
